pkg/tsdb/azuremonitor/types/types.go

```diff
@@ -109,21 +109,26 @@ type AzureResponseTable struct {
 // AzureMonitorJSONQuery is the frontend JSON query model for an Azure Monitor query.
 type AzureMonitorJSONQuery struct {
 	AzureMonitor struct {
-		Aggregation         string  `json:"aggregation"`
-		Alias               string  `json:"alias"`
+		ResourceURI     string `json:"resource"`
```

Comment:
maybe it should be called resourceURI rather than resource? I could see resource being confusing maybe